common/bootstrap/templates/prometheus.yaml

```diff
@@ -74,6 +74,49 @@ spec:
                 static_configs:
                   - targets:
                     - ninechronicles-headless-metrics-aggregator.monitoring.svc.cluster.local:80
+              - job_name: scrape-headlesses
+                metrics_path: /metrics
+                scrape_interval: 1m
+                scrape_timeout: 50s
+                static_configs:
+                  - targets:
+                    - 9c-main-validator-1.nine-chronicles.com:80
+                    - 9c-main-validator-2.nine-chronicles.com:80
+                    - 9c-main-validator-3.nine-chronicles.com:80
+                    - 9c-main-validator-4.nine-chronicles.com:80
+                    labels:
+                      group: validator
+                  - targets:
+                    - 9c-main-rpc-1.nine-chronicles.com:80
+                    - 9c-main-rpc-2.nine-chronicles.com:80
+                    - 9c-main-rpc-3.nine-chronicles.com:80
+                    - 9c-main-rpc-4.nine-chronicles.com:80
+                    - 9c-main-rpc-5.nine-chronicles.com:80
+                    - tky-nc-1.ninodes.com:80
+                    - sgp-nc-1.ninodes.com:80
+                    - nj-nc-1.ninodes.com:80
+                    - la-nc-1.ninodes.com:80
+                    - fra-nc-1.ninodes.com:80
+                    labels:
+                      group: rpc
+                  - targets:
+                    - 9c-main-full-state.planetarium.dev:80
+                    labels:
+                      group: full-state
+                  - targets:
+                    - api.9c.gg:80
+                    - 9c-main-data-provider-db.nine-chronicles.com:80
+                    labels:
+                      group: data-provider
+                  - targets:
+                    - d131807iozwu1d.cloudfront.net:80
+                    labels:
+                      group: explorer
+                  - targets:
+                    - 9c-main-onboarding-query.nine-chronicles.com:80
+                    - 9c-main-onboarding-transfer.nine-chronicles.com:80
+                    labels:
+                      group: onboarding
```

Comment:
we don't use onboarding headless anymore :)

Reply:
I removed the parts, thanks! 🙏🏻